README.md

```diff
@@ -1,21 +1,41 @@
-# Unified InstantSearch E-Commerce
+<div align="center">
+  <h1>Unified InstantSearch for Ecommerce</h1>
+  <p><strong>A ready-to-use, fully functional search interface on top of InstantSearch</strong></p>
+</div>
 
-<p align="center">
-<a href="https://www.algolia.com/doc/guides/building-search-ui/resources/unified-instantsearch/react/"><img src="./.github/unified-instantsearch-ecommerce.png" alt="Unified InstantSearch E-Commerce" width="1280" /></a>
-</p>
+---
 
-**This project aims to be used by any e-commerce website to bootstrap a search experience powered by Algolia the fastest way possible.**
+Unified InstantSearch for Ecommerce combines the most essential widgets and best practices of a great search experience while retaining the customizability of [InstantSearch](https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/js/).
```

Comment:
May I suggest we add a description around those lines?

---
Build your Ecommerce Search & Discovery experience in no time using this framework agnostic prebuilt UI that offers out the box a robust as-you-type InstantSearch experience, while retaining the customizability of [Algolia InstantSearch](https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/react/).

Designed to accelerate your Algolia integration, this mobile optimised UI has been created to best fit any website UI via its full page overlay and offers a direct access to the following Algolia key features:
* Typo proof as-you-type InstantSearch experience
* Query Suggestions
* Advanced Analytics with A/B testing
* Dynamic Re-ranking
* Personalization
* Merchandising capabilities, including banners via Rules

On the frontend side, once compiled the total package size <300kb, and runs on any existing website, regardless of the framework used (React, jQuery, VanillaJS, ...) - see the [browser compatibility](). Its impact on Core Vitals is almost transparent.

Reply:
Done in ac7ea70